projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1359,32 +1382,56 @@ public function process_submission() {
 		 */
 		do_action( 'grunion_after_feedback_post_inserted', $post_id, $this, $is_spam, $entry_values );
 
+		/**
+		 * Filter the title used in the response email.
+		 *
+		 * @module contact-form
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param string the title of the email
+		 */
+		$title   = apply_filters( 'jetpack_forms_response_email_title', __( 'You got a new response!', 'jetpack-forms' ) );
 		$message = self::get_compiled_form_for_email( $post_id, $this );
 
-		array_push(
-			$message,
-			'<br />',
-			'<hr />',
-			__( 'Time:', 'jetpack-forms' ) . ' ' . $time . '<br />',
-			__( 'IP Address:', 'jetpack-forms' ) . ' ' . $comment_author_IP . '<br />', // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
-			__( 'Contact Form URL:', 'jetpack-forms' ) . ' ' . $url . '<br />'
-		);
-
 		if ( is_user_logged_in() ) {
-			array_push(
-				$message,
-				sprintf(
-					// translators: the the name of the site.
-					'<p>' . __( 'Sent by a verified %s user.', 'jetpack-forms' ) . '</p>',
-					isset( $GLOBALS['current_site']->site_name ) && $GLOBALS['current_site']->site_name ?
-						$GLOBALS['current_site']->site_name : '"' . get_option( 'blogname' ) . '"'
-				)
+			$sent_by_text = sprintf(
+				// translators: the the name of the site.
+				'<br />' . __( 'Sent by a verified %s user.', 'jetpack-forms' ) . '<br />',
+				isset( $GLOBALS['current_site']->site_name ) && $GLOBALS['current_site']->site_name ? $GLOBALS['current_site']->site_name : '"' . get_option( 'blogname' ) . '"'
 			);
 		} else {
-			array_push( $message, '<p>' . __( 'Sent by an unverified visitor to your site.', 'jetpack-forms' ) . '</p>' );
+			$sent_by_text = '<br />' . __( 'Sent by an unverified visitor to your site.', 'jetpack-forms' ) . '<br />';
 		}
 
-		$message = join( '', $message );
+		$footer = join(
```

Comment:
Following the conversation in https://github.com/Automattic/jetpack/pull/30517#discussion_r1188387882 , I think we'll want to take this opportunity to move from `join` to `implode` here.

Reply:
Looks like join is used a few times. I'll replace them all.